api/src/schema/Header.js

```diff
@@ -0,0 +1,27 @@
+export const createHeader = i18n => {
+  const Header = `
+    type Header {
```

Comment:
It would be good to add the cache control stuff here as well.

`type Header @cacheControl(maxAge: 90) {`

Reply:
Good idea.